### PR TITLE
Adding builderconfig back, with the dependent code adjusted to use this.

### PR DESCRIPTION
**Pull request by aradhakrishnanGFDL**

### CODE REVIEW

**Comment by ceblanton** [comment]
Strange, I can't add @Ciheim as a reviewer either.. but why?

**Review by Ciheim — APPROVED** [review]

File: intakebuilder/builderconfig.py
[the remark this answers is not in the document]
Comment (aradhakrishnanGFDL) [reply intakebuilder/builderconfig.py:1]: Sure, I am not following the first point as much. Can you make suggestions in GitHub as you review the code using the feature GitHub offers -- to address this (doc or code)?

I will then review, expand and fold in changes!

**Comment by aradhakrishnanGFDL** [comment]
@Ciheim @ceblanton marking this as draft, but please keep reviewing and testing.
Draft - because I see the builderconfig in the location it is present is not a good way for users (through fre-cli or directly) to edit configs. I want to discuss having a --config option instead, with a default option of how it appears in builderconfig today.

File: intakebuilder/builderconfig.py
[the remark this answers is not in the document]
Comment (aradhakrishnanGFDL) [reply intakebuilder/builderconfig.py:1]: @Ciheim changes are complete. over to you for testing and merging in based on the tests.

**Pull request merged**